src/datahike/db.cljc

```diff
@@ -234,7 +234,8 @@
 
        Seqable (seq [db] (dbi/-datoms db :eavt []))
 
-       clojure.lang.ILookup (valAt [db k] (throw (UnsupportedOperationException. "valAt/2 is not supported on FilteredDB")))
+       clojure.lang.ILookup (valAt [db k]
+                                   (throw (UnsupportedOperationException. "valAt/2 is not supported on FilteredDB")))
```

Comment:
Maybe change this back to the original formatting, so this file doesn't get unnecessarily flagged for having changed